apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/TimerInterceptor.java

```diff
@@ -0,0 +1,37 @@
+package gov.cms.bfd.server.war;
+
+import ca.uhn.fhir.interceptor.api.Hook;
+import ca.uhn.fhir.interceptor.api.Interceptor;
+import ca.uhn.fhir.interceptor.api.Pointcut;
+import ca.uhn.fhir.rest.server.servlet.ServletRequestDetails;
+import gov.cms.bfd.server.sharedutils.BfdMDC;
+import java.sql.Timestamp;
+import org.slf4j.MDC;
+
+@Interceptor
+public class TimerInterceptor {
+
+  @Hook(Pointcut.SERVER_INCOMING_REQUEST_PRE_HANDLED)
```

Comment:
Please link to the hapi docs on these various interceptors and pointcuts

Reply:
Updated